cmd/monaco/integrationtest/account/users_test.go

```diff
@@ -19,30 +19,235 @@
 package account
 
 import (
+	"context"
+	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/clients/accounts"
+	accountmanagement "github.com/dynatrace/dynatrace-configuration-as-code-core/gen/account_management"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/featureflags"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account/deployer"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
+	"net/http"
+	"slices"
+	"strings"
 	"testing"
 )
 
 func TestUsers(t *testing.T) {
```

Comment:
Can we rename this to something like "all_resources" on file and test level? (Random name based on all_configs)
This is currently the only test for all account resources, and not a test for configuring users.